src/tests/mocks/hack.rs

```diff
@@ -0,0 +1,114 @@
+use super::*;
```

Comment:
I would rename the file to something like `dogstatsd_client`, even though `hack` is cooler

Reply:
+1